### ISSUE TITLE
Function app: long-running queue trigger runs twice on AppService plan

### ISSUE THREAD

**Issue by knikol** [Azure/azure-webjobs-sdk-script#1587]
**Repro steps**
Step 1: Create function app which is hosted on AppService plan (Paid SKUs) - has no limit on execution time.
Step 2: Create C# queue trigger which runs 7 minutes
**run.csx**
```
using System;

public static async Task Run(string myQueueItem, TraceWriter log)
{
    log.Info($"Start queue trigger: {myQueueItem}");
    await Task.Delay(TimeSpan.FromMinutes(7));
    log.Info($"Completed queue trigger: {myQueueItem}");
}
```
**function.json**
```
{
  "disabled": false,
  "bindings": [
    {
      "name": "myQueueItem",
      "type": "queueTrigger",
      "direction": "in",
      "queueName": "test-trigger",
      "connection": "StorageConnectionString"
    }
  ]
}
```
**host.json**
```
{
  "id": "GUID",
  "functionTimeout": "01:00:00",
  "queues": {
    "maxPollingInterval": 5000
  }
}
```
Step 3: Add "test1" string to the queue which triggers the function

**Expected behavior**
Function executes once: succeeded, no message in the related poison queue.

**Actual behavior**
Function executes twice: succeeded, no message in the related poison queue.
Trace logs:

> 2017-06-09T07:36:25.346 Function started (Id=e7a607b2-c54c-4091-ab25-eb513a242f2d)
> 2017-06-09T07:36:25.346 Start queue trigger: test1
> 2017-06-09T07:43:25.358 Completed queue trigger: test1
> 2017-06-09T07:43:25.358 Function completed (Success, Id=e7a607b2-c54c-4091-ab25-eb513a242f2d, Duration=420012ms)
> 2017-06-09T07:51:27.048 Function started (Id=575f2f6a-5ed5-4148-9d0d-8b3bc3968d81)
> 2017-06-09T07:51:27.048 Start queue trigger: test1
> 2017-06-09T07:58:27.059 Completed queue trigger: test1
> 2017-06-09T07:58:27.059 Function completed (Success, Id=575f2f6a-5ed5-4148-9d0d-8b3bc3968d81, Duration=420005ms)

**Related information**
The issue appeared around 2-3 days ago. Before we did not observe such behavior.

**Comment by knikol** [comment]
BTW, the AlwaysOn has been disabled for the function app. 
[As recommended](https://github.com/Azure/Azure-Functions/wiki/Enable-Always-On-when-running-on-dedicated-App-Service-Plan), enabled it to observe the behavior.

**Comment by knikol** [comment]
Did not help: triggered the second time after ~7 minutes or so.

**Comment by knikol** [comment]
According to the invocation log - trigger goes to infinite loop...
![trigger](https://user-images.githubusercontent.com/27567866/26989612-7247c854-4d5c-11e7-92c3-b8b3042a8b50.png)

**Comment by christopheranderson** [comment]
This is odd. Thanks for the investigation. We'll look into what might have caused this.

**Comment by brettsam** [comment]
related to #1600

**Comment by brettsam** [comment]
Fixed with https://github.com/Azure/azure-webjobs-sdk/pull/1191

**Comment by knikol** [comment]
@brettsam How can I know when the fix is deployed?

**Comment by brettsam** [comment]
In general, watch here for the next release: https://github.com/Azure/azure-webjobs-sdk-script/releases.
But since I've seen multiple people hitting this issue, I'll swing back here and update the related issues when the release is ready.

**Comment by brettsam** [comment]
Closing as it's fixed and we're waiting for it to appear in production.

**Comment by brettsam** [comment]
Update: the fix is now deployed in version 1.0.11015.0 (visible in the 'Settings' page of the Functions portal). If you are not running this version, manually restart your app and it will pick up the latest version.

**Comment by senthil-balunbcuni** [comment]
![image](https://user-images.githubusercontent.com/31780529/51331770-6e2e3400-1a48-11e9-994f-4e49bbe2e31e.png)



I am seeing this issue again. My runtime version is (Runtime version: 1.0.12205.0 (~1))
